sbws/util/fs.py

```diff
@@ -0,0 +1,50 @@
+"""Utils file system functions"""
+
+import logging
+import shutil
+
+log = logging.getLogger(__name__)
+
+
+def sbws_required_disk_space(conf):
+    """Disk space required by sbws files.
+    Rough calculations.
+
+    :param ConfigParser conf: sbws configuration
+    :returns: int, size in MiB
+    """
+    # Number of relays per line average size in Bytes
+    size_v3bw_file = 7500 * 220
+    num_v3bw_files = int(conf['general']['data_period'])
+    # not counting compressed files
+    space_v3bw_files = size_v3bw_file * num_v3bw_files
+    # not counted rotated files and assuming that when it is not rotated the
+    # size will be aproximately 10MiB
+    size_log_file = (int(conf['logging']['to_file_max_bytes']) or 10485760) \
+        if conf['logging']['to_stdout'] == 'yes' else 0
```

Comment:
Shouldn't this be to_file, not to_stdout? Also, you should do `if conf.getboolean('logging', 'to_file')` since there are more was to indicate a true value than the word "yes" https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.getboolean

Reply:
you're right, mistake